userguide/install.rst

```diff
@@ -19,15 +20,52 @@ If you're an experienced android developer and want to help us bring Ubuntu Touc
+    BE VERY CAREFUL! You are responsible for your own actions!
+
+1. Disconnect all devices and non-essential peripherals from your PC. Charge your device on a wall-charger (not your PC) to at least 40 percent
+2. Download the ubuntu touch rom for your device
+
+    - `Bq E4.5 <http://www.mibqyyo.com/de-download/categorias/smartphones/aquaris-e-serie/aquaris-e4-5-ubuntu-edition/>`_ (*krillin*)
+    - `Bq E5 HD <https://storage.googleapis.com/otas/2014/Smartphones/Aquaris_E5_HD/Ubuntu/VEGETA01A-S11A_BQ_L100EN_2017_170207.zip>`_ (*vegetahd*)
+    - `Bq M10 HD <https://storage.googleapis.com/otas/2015/Tablets/Freezer/Ubuntu/OTA_13_FW_version/cooler-image-stable-6.zip>`_ (*cooler*)
+    - `Bq M10 FHD <https://storage.googleapis.com/otas/2015/Tablets/Freezer%20FHD/Ubuntu/OTA_15/frieza-image-stable-8.zip>`_ (*frieza*)
+    - `Meizu MX4 <http://people.canonical.com/~alextu/tangxi/684.zip>`_ (*arale*)
+
+3. Download `SP flash tool <https://storage.googleapis.com/otas/2015/Tablets/Freezer%20FHD/Ubuntu/SP_Flash_Tool_exe_Linux_v5.1612.00.100.zip>`_ for linux.
+
+On Ubuntu 17.10 there are issues with flash_tool loading shared library libpng12 so this can be used as a workaround
+
+.. code-block:: text
+
+    wget -q -O /tmp/libpng12.deb http://mirrors.kernel.org/ubuntu/pool/main/libp/libpng/libpng12-0_1.2.54-1ubuntu1_amd64.deb \
+    && sudo dpkg -i /tmp/libpng12.deb \
+    && rm /tmp/libpng12.deb
+
+You will also need to use `the latest version of the tool <https://spflashtool.com/download/SP_Flash_Tool_v5.1744_Linux.zip>`_.
+
+4. Extract the zip files
+5. Start the tool with sudo
+6. Select the ``*Android_scatter.txt`` file from the archive you downloaded in the first step as the scatter-loading file
+7. Choose "Firmware Upgrade"
+8. Double-check you chose "Firmware Upgrade" and not "Download" or "Format All"
+
+.. warning::
+
+    If select DOWNLOAD rather than FIRMWARE UPGRADE, you will end up with a useless brick rather than a fancy Ubuntu Touch device. You want FIRMWARE UPGRADE.
+
+9. Turn your phone completly off, but do not connect it yet
+10. Perform a final sanity-check that cho selected the "Firmware Upgrade" option, not "Download". Now, press the button labeled "Download"
+11. You selected "Download" instead of just hitting the download button, didn't you? You stupid idiot! But not to worry, that's why we didn't connect your phone yet. Close the tool, make sure you understand the difference between the "Download" option and the "Download" button and start over. If you in fact selected "Firmware Upgrade", you can proceed to the next step
+12. Make sure your phone is off and connect it to your PC. Don't use a usb-3 port, since that's known to cause problems
+13. Magic happens. Your phone will boot into a super old version of Ubuntu Touch
+14. Congratulations, you did not fuck it up! This is good, this is good news. You should not actually use this version, but now you can use the UBports Installer to do the rest
```

Comment:
We're absolutely not swearing in the documentation. Maybe replace this last step with,

"Congratulations! Your device is now running a very old version of Ubuntu for Devices from Canonical. Now use :doc:\`the UBports Installer \<link to other install doc\>\` to upgrade to Ubuntu Touch from UBports."

Reply:
You know, there are times when i doubt your american-ness, but moments like this are very reassuring ;) Ok, i will change it.